x-pack/plugins/canvas/public/services/legacy/stubs/reporting.ts

```diff
@@ -12,6 +12,8 @@ export const reportingService: ReportingService = {
     usesUiCapabilities: () => true,
     components: {
       ReportingPanelPDF: () => (null as unknown) as JSX.Element,
+      ReportingPanelPDFV2: () => (null as unknown) as JSX.Element,
+      ReportingPanelPNGV2: () => (null as unknown) as JSX.Element,
```

Comment:
Canvas currently does not have a PNG reporting integration

Reply:
Gotcha, I added these due to a type issue, perhaps I could just tell TS to ignore that, but it seemed fairly cheap to add the other mock values.